src/oca_github_bot/commands.py

```diff
@@ -53,14 +62,20 @@ def delay(self, org, repo, pr, username, dry_run=False):
 
 
 class BotCommandMerge(BotCommand):
-    bumpversion_mode = None  # optional str: major|minor|patch
+    bumpversion_mode = None
+    bumpversion_mode_list = ["major", "minor", "patch", "nobump"]
 
     def parse_options(self, options):
-        if not options:
-            return
-        if len(options) == 1 and options[0] in ("major", "minor", "patch"):
-            self.bumpversion_mode = options[0]
-        else:
+        # Parse bumpversion
+        res = list(set(options) & set(self.bumpversion_mode_list))
```

Comment:
not sure I understand this line

Reply:
I refactored a little bit the function, with in mind that in a few times, other options will be available. (``squash`` for exemple) optional or required.

So this line is an intersection between available bumversion options and the options of the current call. It will work with ``\ocabot merge patch option_1 option_2`` and also with ``\ocabot merge option_1 option_2 patch``
I see it quite pythonistic, but if you want I can replace by a ``[x for x in options if x in self.bumpversion_mode_list]``.